src/algorithms/tracking/ActsExamples/EventData/GeometryContainers.hpp

```diff
@@ -11,8 +11,8 @@
 #include <boost/container/flat_map.hpp>
 #include <boost/container/flat_set.hpp>
 
-#include "Utilities/GroupBy.hpp"
-#include "Utilities/Range.hpp"
+#include "../Utilities/GroupBy.hpp"
```

Comment:
First and only file with ugly relative include. Fixing more fundamentally with include directories is likely possible too, but this primarily affects so-called benchmarks which may disappear anyway.

Reply:
In the ActsExamples build, this is resolved by a globally adding the ActsExample directory to target_include_directories. In Juggler and EICrecon previously, this was resolved by placing Utilities under EventData (where it does not belong or is distributed by Acts). This moves these files in the right place, but we don't want to add the ActsExamples directory as a target_include_directories for all targets.

The relative include could be avoided, *in a future PR*, by providing the ActsExamples target_include_directory to the targets that link against libalgoriths_tracking through its public interface.